front-end/src/Home.jsx

```diff
@@ -65,7 +66,10 @@ function Home() {
                     </SimpleGrid>
                 )}
             </main>
-        </div>
+            <footer>
+                <Footer />
+            </footer>
```

Comment:
Wondering if we can actually move the `<footer>` tags inside the `<Footer />` component. Would that make more sense? What do you think?

Reply:
> Wondering if we can actually move the `<footer>` tags inside the `<Footer />` component. Would that make more sense? What do you think?

I put it like that because then I can put the semantic tag of footer in my pages, for example, header, main, and then now I have footer. Let me know what you think and if anything I can change it!